example/src/main/scala/example/AuthenticationServer.scala

```diff
@@ -1,13 +1,16 @@
 package example
 
 import pdi.jwt.{Jwt, JwtAlgorithm, JwtClaim}
+import zhttp.http.Middleware.bearerAuth
 import zhttp.http._
 import zhttp.service.Server
 import zio.{App, ExitCode, URIO}
 
 import java.time.Clock
 
-object Authentication extends App {
+object AuthenticationServer extends App {
```

Comment:
same as for client, add documentation at class level.

Reply:
Done